### ISSUE TITLE
NSNotification crash

### ISSUE THREAD

**Issue by yonaskolb** [cbpowell/MarqueeLabel-Swift#24]
I'm getting a new crash

Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '-[MarqueeLabel_Swift.MarqueeLabel observedViewControllerChange:]: unrecognized selector sent to instance 0x7fc8da96a1f0

**Comment by GoodBrue** [comment]
Same here. Even if I comment out all my marquee label scroll code, I still get the exception.

**Comment by cbpowell** [comment]
Not sure how this didn't come up sooner…none of the methods called by the NSNotifications handlers [registered here](https://github.com/cbpowell/MarqueeLabel-Swift/blob/2dbba6671c86bcc07ad4802163977182454dd2e5/Classes/MarqueeLabel.swift#L311-L320) are marked `public`, so you get the unrecognized selector crash.

**Comment by cbpowell** [comment]
Fixing it in a new release shortly - if you want to test it ASAP, try changing the declaration for the observedViewControllerChange function (on line 248) from:

```
func observedViewControllerChange(notification: NSNotification)
```

to:

```
public func observedViewControllerChange(notification: NSNotification)
```

And see if that prevents the crash?

**Comment by cbpowell** [comment]
Fixed this in [release 2.6.1](https://github.com/cbpowell/MarqueeLabel-Swift/releases/tag/2.6.1)! I'll probably make the same changes on the 2.5 and 2.4 branches, since it's kind of a serious problem.
